types/testing-library__jest-dom/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for @testing-library/jest-dom 5.0
+// Project: https://github.com/testing-library/jest-dom
+// Definitions by: Ernesto García <https://github.com/gnapse>
+//                 John Gozde <https://github.com/jgoz>
+//                 Seth Macpherson <https://github.com/smacpherson64>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+/// <reference types="jest" />
```

Comment:
I'm wondering why this needs to be a dependency of `@types/jest`, since Jest ships with its own type declarations now. Can this be removed?

Reply:
This was added prior to Jest 25 being released. AFAIK, Jest 24’s types were not included.

I’d have to look at how Jest’s built-in typings work before giving an answer to your question.